docs/development/README.md

```diff
@@ -94,3 +94,39 @@ If you meet this issue, check following configuration options:
  - `Project Structure -> Project Settings -> Project`, make sure `Project SDK` and `Project Language Level` is set to `17 (Preview) - Pattern matching for switch`.
  - `Project Structure -> Project Settings -> Modules`, make sure that every module has `Language Level` set to `17 (Preview) - Pattern matching for switch (Project default)`  
  - `Settings -> Build, Execution, Deployment -> Build Tools -> Gradle`, make sure that `Gradle JVM` is set to `Project JDK`. 
+
+### Building with docker
+
+![The structure of the docker image](babylon-node-docker-build.png)
+
+
+The application can be build using a single Dockerfile:
+
+```
+docker build . -t radixdlt/babylon-node 
```

Comment:
Can we also use `docker-compose up`?

Reply:
Actually scratch this - I'll let @malexandru-rdx pick this up in his workstream 👍